modules/islandora_google_scholar/islandora_google_scholar.module

```diff
@@ -44,19 +44,22 @@ function islandora_google_scholar_create_meta_tags($object) {
     else {
       return FALSE;
     }
-    foreach ($mods_xml->xpath('mods:name') as $name_xml) {
+    foreach ($mods_xml->xpath(variable_get('islandora_scholar_xpaths_authors_xpath','mods:name')) as $name_xml) {
       $name_parts = array();
       $name_xml->registerXPathNamespace('mods', 'http://www.loc.gov/mods/v3');
-      $roles = $name_xml->xpath('mods:role/mods:roleTerm');
-      $role = strtolower((string) reset($roles));
-      if ($role) {
-        if ($role == 'author') {
+	  //use Mods DisplayForm as a backup source for name
+	  $name_displayForm = $name_xml->xpath('mods:displayForm');
           foreach ($name_xml->xpath('mods:namePart') as $name_part) {
             if ((string) $name_part != '') {
               // Strip periods off the end.
               $np = (string) rtrim($name_part, '.');
               if ($name_part['type'] == 'given') {
-                $name_parts['first_name'] = (string) $name_part;
+				  //middle name is also marked as given name in MODS
```

Comment:
This section is wrongly indented

Reply:
For some reason, the indent show up correctly on my notepad++. Once I pushed it to Github, it is way off. Is there specific thing I need to do to correct it?

Reply:
@dnwk are you using tabs or spaces? Safest to use spaces; they should be consistent everywhere.

Reply:
yes, you need spaces, no tabs

Reply:
Thanks. I am switching to spaces and just ran it through NetBean.